index.html

```diff
@@ -81,10 +81,10 @@ <h3>
           Updates to the Code of Conduct
         </h3>
         <p>
-          The Code of Conduct (CoC) is maintained by the Positive Work Environment Community Group, under delegation from the Advisory Board. In order to keep the CoC up to date with the needs and scope of W3C, PWE will routinely review and update the CoC as needed, as per the <a href="https://www.w3.org/Consortium/Process/#GAProcess">relevant section of the Process</a>. 
+          The Code of Conduct is maintained by the Positive Work Environment Community Group, under delegation from the Advisory Board. In order to keep the Code up to date with the needs and scope of W3C, PWE will routinely review and update the Code as needed, as per the <a href="https://www.w3.org/Consortium/Process/#GAProcess">relevant section of the Process</a>. 
         </p>
         <p>
-          If you have any concerns or issues with the CoC, they can be logged at any time in the <a href="https://github.com/w3c/PWETF/">PWE GitHub repository</a>.
+          If you have any concerns or issues with the code of conduct, they can be logged at any time in the <a href="https://github.com/w3c/PWETF/">PWE GitHub repository</a>.
```

Comment:
It is unclear to me why sometimes you are using lower case and sometimes capitalized words.

* "code of conduct" appears 9 times whereas "Code of Conduct" appears 12 times. Examples:
  > If you have any concerns or issues with the code of conduct,
  
  > Unacceptable behaviors run counter to the Code of Conduct.

- The shorten version "code" appears appears 6 times if I counted correctly whereas "Code" appears 11 times. Examples:
  > The goals of this code are to
  
  >  In order to keep the Code up to date

Reply:
+1 to harmonize the title case "Code of Conduct" everywhere, and to harmonize "Code" everywhere.
After this, I approve the changes! Thank you.

Reply:
Let me go through and normalize the capitalization, thank you for the careful reads!